opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -370,12 +370,14 @@ class ConsoleSpanExporter(SpanExporter):
 
     def __init__(
         self,
+        service_name=None,
```

Comment:
To maintain fuction typing, we could change this to:

```
service_name: Optional[str] = None,
```

Reply:
Thanks for your comments, will do the  changes.